<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorConfigurationDlg</class>
 <widget class="QWidget" name="CalculatorConfigurationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Show controls:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="cbControlMode">
       <item>
        <property name="text">
         <string>Full calculator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Only number input</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Only output</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output mode selection:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbOutputAsk">
     <property name="text">
      <string>Always ask</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbOutputDefault">
     <property name="text">
      <string>Use default without asking</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbAskAndDefault">
       <property name="text">
        <string>Ask, but select default user idles for:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbOutputDefaultTimeout">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>12.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Default mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="cbDefaultOutputMode">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Result</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Calculation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Calculation and result</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formatted result</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formatted calculation and result</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formatted result (money)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formatted calculation and result (money)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbOutputAsk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbOutputAsk</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbDefaultOutputMode</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
